components/viewer/modal/BackpackModal.vue

```diff
@@ -159,11 +163,15 @@ function copyExportText() {
   }, 4000);
 }
 
-const LEGACY_RX = /^(\w+(\/ON#\d+)?)(,(\w+(\/ON#\d+)?))*$/;
+const LEGACY_RX = /([a-zA-Z0-9-]+)(?:\/ON#|:)?(\d*)/g;
```

Comment:
The regex specifically tests the legacy format on the whole string, this new one only searches for one match in the string. 

Here's one that matches the full string: `/^(?:[a-zA-Z0-9-]+(?:\/ON#\d+)?)(?:,[a-zA-Z0-9-]+(?:\/ON#\d+)?)*$/`

https://regex101.com/r/6fSFqi/1